joomla/joomla-cms
# [5.3] Improve inline comment for the "none-ID" behavior within the CRON Scheduler

Pull Request for Issue #43490

### Summary of Changes

Implement "none-ID" usage for the CRON Sheduler

### Testing Instructions

- Setup 5.2-dev
- System -> Scheduled Tasks -> Options -> Disable Lazy Scheduler 
- System -> Scheduled Tasks -> Options -> Enable CRON Scheduler 
- Trigger the CRON Scheduler for example via curl or other external tool
- Notice that nothing will be triggered

### Actual result BEFORE applying this Pull Request

Before the patch the triggered method requires an $id to be passed.

### Expected result AFTER applying this Pull Request

With that patch we allow the usage id as well as "none-ID". With that none-ID thing we trigger one planed task per run.

### Link to documentations
Please select:


### Note to the maintainer

Please double check the #43490 before merging as well as check the upmerge of this PR to 6.0-dev Thanks!
